index.bs

```diff
@@ -426,42 +444,42 @@ Report the layout shift sources {#sec-report-layout-shift-sources}
 When asked to <dfn>report the layout shift sources</dfn> for an active
 {{Document}} |D|, run the following steps:
 
-1. Let |C| be an empty sequence of {{LayoutShiftAttribution}} objects.
+1. Let |C| be an empty <a>sequence</a> of {{Node}} objects.
```

Comment:
I think this should be a list, not a sequence.

Reply:
Done.